### PR TITLE
Splice

### LINKED ISSUE #266: <p> html tags on https://labs.hyperledger.org/labs/splice.html
Page https://labs.hyperledger.org/labs/splice.html displays few `<p>` html tags that shouldn't be there.

321784530-b8f19b8b-e86c-48cb-8ffe-23203db5cfee

### PR DESCRIPTION
**Pull request by simonletort-da**
remove html tags from slice.md markdown file
closes #266

### CODE REVIEW

File: labs/splice.md [hunk 1]
```diff
@@ -3,81 +3,111 @@ layout: default
 title: Splice
 parent: Labs
 ---
-# Splice
+# Hyperledger Splice
```
Comment (tkuhrt) [inline labs/splice.md:6]: Labs cannot be named liked this. Please see [Labs Representation](https://labs.hyperledger.org/proposal.html#labs-representation) for more information. Please update this pull request to remove the "renaming" of the lab.

Comment (simonletort-da) [reply labs/splice.md:6]: Done

Comment (simonletort-da) [reply labs/splice.md]: Hi @tkuhrt, sorry for that, my mistake. fixed.

File: labs/splice.md [hunk 2]
```diff
@@ -21,63 +21,93 @@ Daml is a platform built by Digital Asset designed for deploying blockchains and
...
 
 ## Architecture
-The following diagrams show how the Splice applications interact with the Daml platform from Digital Asset.<br>
+The following diagrams show how the Splice applications interact with the Daml platform from Digital Asset.
 
-![SV Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-1.jpg)
+![SV Node Architecture](./images/splice/Super%20Validator%20Architecture%20Diagram%20for%20Hyperledger%20Splice%20(2).jpg)
 
-![Validator Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-2.jpg)
+![Validator Node Architecture](./images/splice/Super%20Validator%20Architecture%20Diagram%20for%20Hyperledger%20Splice%20(3).jpg)
```
Comment (tkuhrt) [inline labs/splice.md:106]: These images do not exist in the repo. Do they need to be added and the other images removed?

Comment (simonletort-da) [reply labs/splice.md]: fixed image url. sorry (again...)

Comment (simonletort-da) [reply labs/splice.md]: @tkuhrt fixed.

**Review by tkuhrt — APPROVED** [review]

**Review by nidhi-singh02 — APPROVED** [review]
LGTM!

**Pull request merged**